app/components/forms/wizard/attendee-step.js

```diff
@@ -1,6 +1,10 @@
 import Component from '@ember/component';
 import FormMixin from 'open-event-frontend/mixins/form';
 
+function getIdentifier(name, fields) {
```

Comment:
Codacy found an issue: ['fields' is defined but never used.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5730106)